### PR TITLE
Update service-licensing-for-cd.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

**Review by SushrutHarness — DISMISSED** [review]
LGTM

File: docs/continuous-delivery/get-started/service-licensing-for-cd.md [hunk 1]
```diff
@@ -8,31 +8,25 @@ helpdocs_is_private: false
...
 ## Licensing FAQ
 
 ### What are Active Services?
 
-- When Harness deploys a service via a pipeline, the service is counted as an Active Service.
-- When Harness deploys a GitOps Application, the GitOps Application is counted as an Active Service.
-- It remains an Active Service for the next 30 days.
-- Harness obtains all services that are part of any pipeline execution (deployment) over the past 30 Days. This represents the total Active Services consumed by an account.
-- There is no cap on the number of Service Instances (pods, hosts, etc.)  that are created by the service.
+- When Harness deploys a service via a pipeline, the service is counted as an Active Service, in the last 30 days. Harness obtains all services that are part of any pipeline execution (deployment) over the past 30 Days. This represents the total Active Services consumed by an account.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/get-started/service-licensing-for-cd.md:19]:
```suggestion
- When Harness deploys a service via a pipeline, the service is counted as an Active Service if the service is deployed not more than 30 days ago. Harness obtains all services that are part of any pipeline execution (deployment) over the past 30 Days. This represents the total Active Services consumed by an account.
```

File: docs/continuous-delivery/get-started/service-licensing-for-cd.md [hunk 2]
```diff
@@ -8,31 +8,25 @@ helpdocs_is_private: false
...
+- When Harness deploys a GitOps Application, the GitOps Application is counted as an Active Service.e
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/get-started/service-licensing-for-cd.md:20]:
```suggestion
- When Harness deploys a GitOps Application, the GitOps Application is counted as an Active Service.
```

File: docs/continuous-delivery/get-started/service-licensing-for-cd.md [hunk 3]
```diff
@@ -86,7 +88,7 @@ Services types that are calculated this way:
 
 
 ### Serverless Applications
-For serverless applications deployed through a Deploy Stage in a Harness pipeline, for each serverless function that is deployed in the last month, 0.16 (1/6) Active Service Licenses are consumed. We will always round up to the next whole number. For example: 0.2 will become 1 Active Service License. 
+For serverless applications deployed through a Deploy Stage in a Harness pipeline, for each serverless function that is deployed in the last month, 0.16 (1/6) Active Service Licenses are consumed. We will always round up to the next whole number. For example: 0.2 will become 1 Active Service License. For every
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/get-started/service-licensing-for-cd.md:91]:
```suggestion
For serverless applications deployed through a Deploy Stage in a Harness pipeline, for each serverless function that is deployed in the last month, 0.16 (1/6) Active Service Licenses are consumed. We will always round up to the next whole number. For example: 0.2 will become 1 Active Service License. For every service types that are classified as a serverless application:
```

File: docs/continuous-delivery/get-started/service-licensing-for-cd.md [hunk 4]
```diff
@@ -86,7 +88,7 @@ Services types that are calculated this way:
...
 
 Service types that are classified as a serverless application:
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/get-started/service-licensing-for-cd.md:93]:
```suggestion

```

**Review by roshnisarangadharan — APPROVED** [review]
Approved with updates

File: docs/continuous-delivery/get-started/service-licensing-for-cd.md [hunk 5]
```diff
@@ -8,31 +8,27 @@ helpdocs_is_private: false
 helpdocs_is_published: true
 ---
 
-Harness uses a Service usage model to charge Harness Continuous Delivery (CD) module customers.
+Harness Continuous Delivery and GitOps use a Service usage model to track your license consumption.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/get-started/service-licensing-for-cd.md:11]:
```suggestion
Harness Continuous Delivery and GitOps use a service usage model to track your license consumption.
```

File: docs/continuous-delivery/get-started/service-licensing-for-cd.md [hunk 6]
```diff
@@ -8,31 +8,27 @@ helpdocs_is_private: false
...
+Go to **Account Settings** > **Subscriptions** or enter the following URL using your Harness account ID for `ACCOUNT_ID`.
 
-```
-https://app.harness.io/ng/account/ACCOUNT_ID/settings/subscriptions
-```
-
-<DocImage path={require('./static/7998c6375586533d4f1c0192d27bff9d1c82e2311a007a4ad2bf7f7a37dd7da9.png')} width="60%" height="60%" title="Click to view full size image" />  
+#### Key things to understand
 
+- Under License Usage, the Service License tile is the total number of service licenses consumed by all the total active services over the last 30 day period.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/get-started/service-licensing-for-cd.md:29]:
```suggestion
- Under License Usage, the Service License tile shows the total number of service licenses consumed by all the total active services over the last 30 days. 
```

File: docs/continuous-delivery/get-started/service-licensing-for-cd.md [hunk 7]
```diff
@@ -8,31 +8,27 @@ helpdocs_is_private: false
...
-```
-https://app.harness.io/ng/account/ACCOUNT_ID/settings/subscriptions
-```
-
-<DocImage path={require('./static/7998c6375586533d4f1c0192d27bff9d1c82e2311a007a4ad2bf7f7a37dd7da9.png')} width="60%" height="60%" title="Click to view full size image" />  
+#### Key things to understand
 
+- Under License Usage, the Service License tile is the total number of service licenses consumed by all the total active services over the last 30 day period. 
+- The Active Service Graph shows the breakdown of active services consumed over the last 30 days.
+- The Usage Breakdown Table shows the type of services consumed over the last 30 days based on the Active Service Graph
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/get-started/service-licensing-for-cd.md:31]:
```suggestion
- The Usage Breakdown Table shows the type of services consumed over the last 30 days based on the Active Service Graph.
```

**Review by roshnisarangadharan — APPROVED** [review]

**Pull request merged**